### DISCUSSION TITLE
Electrostatic potential

### DISCUSSION THREAD

**Discussion by hanahg** [MDAnalysis/mdanalysis#4428] [Q&A]
Hi,
I want to calculate the electrostatic potential of a nano particle. I could not find which keys should I use for.
Please share with me if there is any discussion.

**Comment by PicoCentauri** [comment]
That is a very interesting task.

First of all the question is how do define your electrostatic potential? The term electrostatic potential usually refers to the double integral of the charge density. However, in the literature the term it is also used for the electrokinetic 𝜻 potential which is a bit harder to calculate and requires the knowledge of the viscosity, velocities and the dielectrics.
